frontend/src/style/vars.scss

```diff
@@ -28,3 +28,5 @@ $black: hsl(0, 0%, 4%);
 // Custom variables for fullheight Map Viewer, same as Bulma defaults
 $navbar-height: 4rem;
 $footer-height: 3.5rem;
+
+$viewer-height: 450px;
```

Comment:
Should we perhaps add a newline?

Reply:
Ah you mean at the end of the file? The linter does not seem to complain as far as I can see.
Another idea would be to add a comment explaining what this variable is for.

Reply:
Github made a red remark (hidden by these discussion I think), not a major thing obv. but maybe nice to fix. Adding a comment is a good idea!